README.md

```diff
@@ -156,8 +154,62 @@ Here is a screenshot of the map-structure in our project:<br><br>
 
 ## 🔨 Install
 
+Here's a manual for installing this project.
+
+### 📡 Install Node.js & npm
+
+To start off, install Node.js & npm. Node.js is a platform built on Chrome's JavaScript runtime for easily building fast and scalable network applications. Npm is a Node Package Manager, to install and manage packages.
+
+To install Node.js and npm, go to <a href="https://nodejs.org/en/">https://nodejs.org/en/</a>, and download the latest version of Node.js on your device. Make sure to select the npm-install option while installing Node.js.
+
+### 📍 Clone the GitHub repository
+
+To get the project on your code-editor, you have to clone it from our GitHub repository. (<a href="https://github.com/veerleprins/FemAlert-meesterproef">https://github.com/veerleprins/FemAlert-meesterproef</a>)
+<br>
+To do this, type the following command in your terminal:
+<br><br>
+`git clone https://github.com/veerleprins/FemAlert-meesterproef.git`
+<br>
+
+### 📦 Install all the used packages
+
+To install all the used packages in this project, type the following command in your terminal:
+<br><br>
+`npm i`
+<br>
+
+### 🟢 Start and run the server
+
+To run the server, type the following command in your terminal:
+<br><br>
+`npm run dev`
+<br>
+
 ## 📋 Sources
 
 https://bradfrost.com/blog/post/atomic-web-design/
 
 ## © License
+
+This repo is licensed as MIT by Veerle Prins, Sharon Veldman and Johan van Beusekom, 2021.
+<br><br><br>
+
+MIT License
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
I believe you don't have to include the full license, this license file is also in the repo. You only have to mention that the project is under mit license and our names

Reply:
Thanks :)
Removed it!